set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
I was told to just use 5.0.0 for everything, and if that's the case there's not much reason to add it to the docker labels stuff (I was planning to just add git-ref info), and there's not much reason for this file. I'm not opposed to changing to using real version numbers again, but I wasn't around for whatever discussion led to the 5.0.0 decision in the first place so I didn't know that was relatively new/potentially malleable as a policy.